### ISSUE TITLE
PHP Warning for NP Technorati Tags

### ISSUE THREAD

**Issue by sajxe** [NucleusCMS/NucleusCMS#57]
I am using the latest version of NP Technorati Tags downloaded from github.
I think I have seen this warning for Nucleus 3.70, PHP 5.6 and php 7.0.

PHP Warning:  ini_set() has been disabled for security reasons in /home/user/nucleus/plugins/php-delicious/php-delicious.inc.php on line 52

I am not sure if this is a problem with my server settings or with the plugin. The plugin still seems to be working ok but I'm just posting it here in case it is something that may need to be fixed later.

Thanks very much!

**Comment by yama** [comment]
https://github.com/NucleusCMS/NP_TechnoratiTags
maybe fixed.

**Comment by sajxe** [comment]
I think this is fixed now (as mentioned in my other posts here https://github.com/NucleusCMS/NucleusCMS/issues/56 ) but I can't check yet because my blog has no display right now :) Later when I can check it I will update and close this issue if it fixed :)

**Comment by sajxe** [comment]
In v. 3.80dev: The tags are displaying at the bottom of the posts, and I can click on them to see a list of posts, but if I try to go to the tag cloud page it says Error No Such Page. It was working ok in the previous version of Technorati Tags.
In the skins page I see a new link "(Change to special skin page)" next to my "tagcloud" Special skin part. I was not sure what that was for so I have not tried to click it yet.
Am I getting the error because I need to change the special skin page in 3.80dev?

**Comment by sajxe** [comment]
Hmm, I tested this on three special skin parts. If I try to click on a link to a special skin part, such as 
`https://test.domain.com/index.php?special=tagcloud`
Nucleus will display the message "Error No Such Page".
This is not a browser error message, it is an error message from Nucleus.
I tried clicking on the new "Change to special skin page" option in 3.80dev to see what would happen.
The special skin part seems to be successfully converted to a special skin page, but it does not fix the error.
In 3.70 there is no problem. The php log for 3.80dev shows the same kinds of errors as I saw in the previous logs I posted in the other threads, nothing different.
